contracts/deploy/mainnet/119_simple_harvester_v2.js

```diff
@@ -0,0 +1,139 @@
+const addresses = require("../../utils/addresses");
+const { deploymentWithGovernanceProposal } = require("../../utils/deploy");
+
+module.exports = deploymentWithGovernanceProposal(
+  {
+    deployName: "119_simple_harvester_v2",
+    forceDeploy: false,
+    //forceSkip: true,
+    reduceQueueTime: true,
+    deployerIsProposer: false,
+    proposalId: "",
+  },
+  async ({ deployWithConfirmation }) => {
+    const { multichainStrategistAddr } = await getNamedAccounts();
+
+    // ---- Menu ---
+    // 1. Deploy new simple Harvester
+    // 2. Change harvester on all SSV strategies
+    // 3. Support strategies on the harvester
+    // 4. Sending all WETH from the OldDripper to FixedRateDripper
+    // 5. Update harvester on OETH AMO
+    // 6. Support AMO strategy on new harvester
+    // --------------
+
+    // 1. Deploy new simple Harvester
+    const cOETHFixedRateDripperProxy = await ethers.getContract(
+      "OETHFixedRateDripperProxy"
+    );
+    const dOETHHarvesterSimple = await deployWithConfirmation(
+      "OETHHarvesterSimple",
+      [
+        addresses.mainnet.Timelock,
+        multichainStrategistAddr,
+        cOETHFixedRateDripperProxy.address,
+        addresses.mainnet.WETH,
+      ],
+      undefined,
+      true
+    );
+    const cOETHHarvesterSimple = await ethers.getContractAt(
+      "OETHHarvesterSimple",
+      dOETHHarvesterSimple.address
+    );
+
+    // 2. Change harvester on all SSV strategies
+    const cNativeStakingStrategy_1 = await ethers.getContractAt(
+      "NativeStakingSSVStrategy",
+      "0x34eDb2ee25751eE67F68A45813B22811687C0238"
```

Comment:
nit: Would be nice if we can read these addresses from the artifacts by using their proxy contract names

Reply:
Fixed in this commit: https://github.com/OriginProtocol/origin-dollar/pull/2363/commits/d0978ab047ca17471ad8ff131ccef43568d55ba3.